fix_checkpoint_history.py

```
@@ -0,0 +1,80 @@
+    csv_path = Path(target_csv)
+    if not csv_path.exists():
+        print(f"Error: CSV file {target_csv} not found!")
+        return
+    
+    file_size_mb = csv_path.stat().st_size / 1024 / 1024
+    print(f"Found CSV file: {target_csv}")
+    print(f"File size: {file_size_mb:.1f} MB")
+    
+    # Count lines to estimate progress
+    print("Counting records...")
+    with open(csv_path, 'r') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)